server.go

```diff
@@ -30,8 +31,12 @@ func getDB() (db *sql.DB) {
 }
 
 func main() {
-	http.HandleFunc("/", handleIndex)
-	http.HandleFunc("/steam/discounts", handleSteamDiscounts)
+    mux := mux.NewRouter()
+	mux.HandleFunc("/", handleIndex)
+	mux.HandleFunc("/steam/discounts", handleSteamDiscounts)
+	mux.HandleFunc("/steam/featured/{feature:[a-z]+}", handleSteamFeatured)
```

Comment:
in that case, how about GET parameter? but yeah, gorilla's mux seems better than original go http router